src/drivers/tap_esc/tap_esc.cpp

```diff
@@ -262,6 +262,9 @@ TAP_ESC::init()
 		return ret;
 	}
 
+#ifdef CONFIG_ARCH_BOARD_AEROFC_V1
```

Comment:
I can add a comment explaining and leaving a TODO. It doesn't make much sense... but at least it allows to fly with our version of the hardware.